experimental/packages/opentelemetry-instrumentation/test/node/EsmInstrumentation.test.mjs

```diff
@@ -18,7 +18,7 @@ import * as assert from 'assert';
 import {
   InstrumentationBase,
   InstrumentationNodeModuleDefinition,
-} from '../../build/src/index.js';
+} from '../../build/src/platform/index.js';
```

Comment:
This change WAS required for me to be able to build and test locally (on Windows), not exactly sure "why" the `tsconfig.json` or `package.json` changes caused this... Perhaps, this was just a general windows verison of mocha issue???

Reply:
@JamieDanielson do you maybe know why this is needed? :thinking:

Reply:
My best guess is that this is to do with how mocha deals with imports, I know that there are some differences with mocha on windows systems so it might not be reading `package.json` correctly to infer which platform it needs to come from.